tests/acceptance/init.spec.js

```diff
@@ -53,7 +53,7 @@ describe('Acceptance: ng init', function() {
         });
 
         expected.forEach(function (file, index) {
-            expected[index] = file.replace(/__name__/g, 'angular-cli');
+            expected[index] = file.replace(/__name__/g, 'tmp');
```

Comment:
I was struggling with getting the tests to pass when I was bumping versions. I see you changed the expectation, does that mean that the API for the ember-cli changed?